### DISCUSSION TITLE
Linux Support / Desktop application / Pinephone

### DISCUSSION THREAD

**Discussion by ghost** [gsantner/markor#1539] [Ideas]
## General information
* App version: 2.8.4
* Android version: LineageOS 18.1 (Android 11)
* Device: Moto G7 Play
* I understand that this issue will be closed without further notice in case I didn't describe the problem including general information, or didn't [search](https://github.com/gsantner/markor/issues?q=) for similar issues & [requests](https://github.com/gsantner/markor/discussions?discussions_q=).

### Description

Can you add support for Linux and Linux mobile devices like the Pinephone Pro? https://www.pine64.org/pinephonepro/

**Comment by gsantner** [comment]
![image](https://user-images.githubusercontent.com/6735650/147297003-1384b9cd-f520-4398-a173-87130c821111.png)

So you have installed some Android ROM ... on a Pinephone, an want to install Markor there -- do I understand correct?  
I don't quite get it why you talk about Motorola there then.

I don't know about the PinePhone's Android state and what works at that Android ROM yet. Markor doesn't use any special google libraries or  - so should work fine out of the box without any gapps. If however that Android ROM is borked, you may be better off contacting the people working on the ROM. 

And no, this is a real Android app that uses Android API's - it requires Android. You need to look for an alternative to Markor for generic desktop/mobile GNU/Linux if you don't have Android (OS/Anbox/Emulator).

**Reply by ghost** [reply]
I don't currently have a Pinephone Pro, but I'm planning on getting one in the near future, so I'm hoping that most of the apps I currently use may work on the Pinephone Pro by the time I get one.

**Reply by gsantner** [reply]
When run Android on it, Markor will work.

If it makes sense to buy a Pinephone, just to install Android again - is up to you.  
For Ubuntu touch for example there is a simple markdown editor.

> I'm hoping that most of the apps I currently use may work on the Pinephone Pro by the time I get one

You can already prepare for big changes if you plan to use a GNU/Linux OS (non android) as daily driver. I'm quite sure Pinephone won't fit for you if you can't embrace changes, but want to stick to what you are used to ("just under a different coat"). 

Just leaving this here. <https://github.com/GloDroid/glodroid_manifest/issues>

**Reply by ghost** [reply]
I just hope that there will at least be alternatives to all of the apps I currently use.

**Reply by gsantner** [reply]
>  to all of the apps I currently use.

There is an alternative for many things, but for sure not all things. Especially not banking/money apps, WhatsApp & AAA games. Many things are only for Android & iOS.

I have a PinePhone on my own, but I'm/were fully aware of this all. Just giving you the advice to not spend lots of money if you are not.

**Reply by ghost** [reply]
I don't use banking apps and almost all of my apps are FOSS. I think the only app I have that isn't FOSS is a local news app.
